src/main/resources/com/adobe/epubcheck/schema/20/rng/opf20.rng

```diff
@@ -168,12 +166,9 @@ Authors:
       <data type="ID"/>
     </attribute>
     </optional>
-    <ref name="OPF20.optional-xsi-type"/>
+
     <ref name="OPF20.optional-scheme-attribute"/>
-    <!-- <ref name="DC.metadata-common-content"/>
-    MG 20111024, see epubcheck Issue 114
-    -->
-    <ref name="DC.metadata-required-content"/>    
+    <ref name="DC.metadata-common-content"/>
```

Comment:
It appears that the schema from http://www.idpf.org/epub/20/spec/OPF_2.0_latest.htm#AppendixA allows empty dc:identifier elements.  This is because 


```
<define name="DC.metadata-common-content">
  <text/>
</define>
```
allows any text including the empty string.